### ISSUE TITLE
cannot change port for dev

### ISSUE THREAD

**Issue by rcholic** [mxstbr/react-boilerplate#462]
Thanks for the great starter kit!

When I was running `npm start -- -p 5000`, the app still runs on the default port 3000. I also tried `npm start -p 5000`, but got same result. So I was wondering if this has been an issue or it is just me who runs into this (?).

**Comment by kunal-mandalia** [comment]
I don't think many people are running dev on a different port ... do you want to run it on 5000 as you're running another server on 3000?

In server/index.js you could always do something like:

```
const port = process.env.PORT || 3000 // default
if (isDev) {
  port = 5000;
}
```

**Comment by mxstbr** [comment]
Ah yes, good catch @rcholic, that's a bug, that feature got lost in the command restructuring! Would you mind submitting a PR to fix this? Thanks!

**Comment by rcholic** [comment]
@mxstbr I will be more than happy to make a PR. Will come back soon! Thanks

**Comment by mxstbr** [comment]
Fixed by @rcholic in #505! :tada: Thanks so much!
